common/patches/ss_define.h

```diff
@@ -70,6 +70,9 @@
 #define OUT_str(x) \
 	strncpy(eq->x, emu->x, sizeof(eq->x)); \
         eq->x[sizeof(eq->x)-1] = '\0';
+#define OUT_str_x_y(x,y) \
```

Comment:
This looks unused, not sure if intended to use it somewhere or leftover from testing